src/tests/dictionaryConcepts/reducer/dictionaryConcept.test.js

```diff
@@ -36,7 +36,7 @@ beforeEach(() => {
   state = {
     concepts: [],
     loading: false,
-    dictionaryConcepts: [],
+    dictionaryConcepts: [paginatedConcepts.concepts],
```

Comment:
Is this part of the ticket?

Reply:
Yes, it is. It caters for the updated reducer.